joboffers/forms.py

```diff
@@ -68,3 +71,35 @@ class Meta:
             'salary': '',
             'description': 'Descripción de la oferta'
         }
+
+
+class TagSearchSelect2(autocomplete.Select2):
```

Comment:
Please add docstrings to classes and functions so we don't need to guess what they do.

Reply:
Added